scripts/scons_plugin/osutils.py

```diff
@@ -116,10 +116,24 @@ def _which(env, prog, mode, searchpath):
 
     return paths
 
+# Find and return path to the package (its install prefix directory) of the latest version
+# of `pkg_name` package installed by brew. If not found, None is returned.
+def FindBrewPackage(env, pkg_name, required=True):
+    if not env.Which('brew'):
+        return None
+
+    # $(brew --prefix)/opt/some_pkg should be a link to the latest version of some_pkg,
+    # e.g. ../Cellar/some_pkg@1.2/1.2.3/
+    prefix = os.path.join(env.GetCommandOutput('brew --prefix'), 'opt', pkg_name)
+    if not os.path.isdir(prefix):
```

Comment:
`GetCommandOutput` can return none (if brew fails, e.g. problem with permissions). We should handle it, otherwise os.path.join will raise exception.

Reply:
done